### ISSUE TITLE
Preferences quick filtering for available docsets

### ISSUE THREAD

**Issue by a-b** [Thibaut/devdocs#644]
To improve docset management experience I'm recommending consider adding a quick filter.

**Comment by Thibaut** [comment]
What do you mean by "quick filter"?

**Comment by j-f1** [comment]
Currently, when you’re looking for docs to enable, you have to scroll through the list. It would be much easier if there was a textbox to filter the docset names to find the one you want.

**Comment by Thibaut** [comment]
Right. All the docs are indexed and can be enabled in the main search:

![devdocs api documentation 2017-06-30 16-34-51](https://user-images.githubusercontent.com/17579/27753065-1dbae3dc-5db2-11e7-9713-7e909a95cfe5.png)

Adding another search on the preference page would be a lot of work. I'm not sure it's worth it.
